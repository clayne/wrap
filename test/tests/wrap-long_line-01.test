wrap | /dev/null | | long_line-01.txt | 0
