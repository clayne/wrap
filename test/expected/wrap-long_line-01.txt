X
ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789ABCDEFGHIJKLMNOPQ
